ui/app/components/job-status/deployment-history.js

```diff
@@ -71,4 +72,25 @@ export default class JobStatusDeploymentHistoryComponent extends Component {
       color: 'critical',
     });
   }
+
+  // #region search
+
+  /**
+   * @type { string }
+   */
+  @tracked searchTerm = '';
```

Comment:
`question`: I noticed that this is not associated to a query parameter, as you've requested in previous PRs. Any color about why we're not using an associated query parameter here?

Reply:
Good question! In this case, this is a conditional component — it only exists when a deployment is active (a much smaller % of the time than when it's in steady state).

If I wanted to add a queryParam that controlled this from the job index route, there would be a majority of time where that queryParam did nothing.

There's an argument to be made for including it anyway, but it seems pretty outweighed by the argument that each deployment is separate and search terms aren't necessarily something you'd want to maintain between all of them.

Reply:
Gotcha!